lib/mongo/cursor.rb

```diff
@@ -294,6 +291,8 @@ def close
       end
 
       nil
+    rescue Error::OperationFailure, Error::SocketError, Error::SocketTimeoutError => e
```

Comment:
```suggestion
    rescue Error::OperationFailure, Error::SocketError, Error::SocketTimeoutError
```
Just so that we don't get a warning about an unused variable